scss/standalone/patterns_icons-application.scss

```diff
@@ -0,0 +1,8 @@
+@import '../settings';
```

Comment:
Because this file is different from out other standalone patterns please include some comment on top that says what it it and where it is needed.

Something like:

"This file contains styles needed to render application icons only. By itself it's is not a standalone pattern file, but is used to display these icons in our examples".